netlify/functions/generate-description.ts

```diff
@@ -0,0 +1,51 @@
+import { Handler, HandlerEvent, HandlerContext } from '@netlify/functions'
+import { Configuration, OpenAIApi } from "openai";
+
+const configuration = new Configuration({
```

Comment:
try moving this into the actual handler function. If you dfine it outside the lambda, you will be missing the env context to get the api key from Netlify

Reply:
Ok! done